Server/src/shutdown.cpp

```diff
@@ -28,11 +23,7 @@
 #  include <windows.h>
 # endif
 
-volatile unsigned int  g_ShutDown = 0;
-
-#ifdef BACKEND_OPEN62541
-UA_Boolean g_RunningFlag = 1;
-#endif
+OpcUa_Boolean g_RunningFlag = 1;
```

Comment:
Should the running flag be atomic? [This stackoverflow thread](https://stackoverflow.com/questions/16111663/do-i-have-to-use-atomicbool-for-exit-bool-variable?noredirect=1&lq=1) seems to suggest so.

Reply:
Sounds fair enough